releasenotes/notes/ecr_cal_builder-358017785a3e296a.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
```

Comment:
This is probably a feature note rather than an upgrade note. We typically reserve upgrade notes for changes that may require a user to make a manual change when upgrading to the new version. This reads more like the `RZXBuilder` pass now supports ecr too in addition to the cx it supported in the past

Reply:
I'll amend. Thanks

Reply:
Done here: [f3b8329](https://github.com/Qiskit/qiskit-terra/pull/9467/commits/f3b83297f0f2f65654d21f1ee0dc89bba23a23e1)